### ISSUE TITLE
Patcher Request - Unique Region Names

### ISSUE THREAD

**Issue by somohexuals** [Synthesis-Collective/Development#29]
Would this be possible? It renames the "to Skyrim" from doors to the hold/city name. EX: Exiting a door from Dragonsreach would say "to Whiterun". Exiting a door from the Whiterun Stables would say "to Whiterun Hold"

Thanks. :)

**Comment by halgari** [comment]
Seems completely possible., I'll take a look this weekend.

**Comment by mrudat** [comment]
From one of the existing mods that do this, a tricky thing is that many of the outdoor cells don't have meaningful names; to do this Right, you'd probably need to have a mod that gives names to the outdoor cells.

**Comment by halgari** [comment]
I agree, I took a look at I'm going to have to do more research before I know where to start with this one.

**Comment by MacSplody** [comment]
Duplicate of https://github.com/Synthesis-Collective/Development/issues/7

**Comment by MacSplody** [comment]
> From one of the existing mods that do this, a tricky thing is that many of the outdoor cells don't have meaningful names; to do this Right, you'd probably need to have a mod that gives names to the outdoor cells.

I already do that with [Unique Region Names Extended](https://www.nexusmods.com/skyrimspecialedition/mods/41853), but its regions don't cover every cell.
It takes time to find the new doors, and then make sure they are connected to an interior.
Once I've found the new door, and know which cell it is in, and that cell isn't inside an existing URN or URN Extended region.
I'll add the area that cell covers to a new or existing region.

**Comment by MacSplody** [comment]
It might actually be better to have a cell region patcher.
There are other mods like [Worldspace Transition Tweaks](https://www.nexusmods.com/skyrimspecialedition/mods/48889), Beyond Skyrim, Open Cities and Cutting Room Floor (to name a few) that also add new regions to cells.
Region changes to cells can be merged, the winner between regions is decided by the data a region changes and its priority.

**Comment by radj307** [comment]
I'm actually working on something that does this at the moment for a personal request.

Currently everything is functional and works well, however the region records specifically are getting mangled when written out.
